### DISCUSSION TITLE
Yolo .onnx can not be handle

### DISCUSSION THREAD

**Discussion by timimm** [zkonduit/ezkl#801] [General]
Hi guys. I wonder if you have the situation that executing `ezkl gen-setting -M yolov8n-face.onnx` with a yolo onnx model file. the Error was: 
<img width="1457" alt="image" src="https://github.com/zkonduit/ezkl/assets/48613938/eb3a3ec5-932a-4d39-a32f-288a094a2270">

Can anyone help me? I want to prove a humanbeing check model with the yolov8n-face model.....

**Comment by alexander-camuto** [comment]
what version of ezkl are you using ?

**Reply by timimm** [reply]
Both the CLI and pip package are the latest, v11.0.14.

**Reply by alexander-camuto** [reply]
can you post a link to the model here :)

**Reply by timimm** [reply]
Yep, certainly!
https://github.com/akanametov/yolov8-face

It needs a transformation, here is the code:
```python
from ultralytics import YOLO

# Load a model
model = YOLO(r"E:\download\yolov8n_100e.pt")  # load an official model
# model = YOLO('path/to/best.pt')  # load a custom trained

# Use the model
success = model.export(format="onnx",simplify=True,opset=12)
```

**Reply by timimm** [reply]
So, is this a bug or not? Anyone help plz :)

**Comment by alexander-camuto** [comment]
This should be patched in part now
